cogs/mod.py

```diff
@@ -48,8 +48,8 @@ def __init__(self, bot):
         """
         dest = user
         await dest.send(
-            content='{}\n*To answer write* `{}answer "your message here"`'
-            .format(message, self.bot.config.command_prefix))
+            content='{}\n*To answer write* `{}answer "your message here"`'.
```

Comment:
Is this a style change for yapf? It should not be breaking the build since it was passing before right? 

If its required then keep it, otherwise keep style the way it was before to keep a clean blame.

Reply:
I had been YAPFing with a different installation from the one installed in the repo by mistake, so this is fixing that incorrect YAPFing.

Reply:
Alright, great!